test/algorithms/simulation/sidb/operational_domain.cpp

```diff
@@ -1358,3 +1383,80 @@ TEMPLATE_TEST_CASE("Grid search to determine the operational domain. The operati
         }
     }
 }
+
+TEST_CASE("Bestagon AND gate operational domain and temperature computation, using cube coordinates",
+          "[operational-domain]")
```

Comment:
We say `cube` coordinates here but it looks like SiQAD coordinates are used

Reply:
Good catch!